src/ensembl/src/global/globalActions.ts

```diff
@@ -1,12 +1,10 @@
 import { createAction } from 'typesafe-actions';
 
 import { BreakpointWidth } from './globalConfig';
-import { getGlobalAnalyticsObject } from '../analyticsHelper';
 
 export const updateBreakpointWidth = createAction(
   'browser/update-breakpoint-width',
   (resolve) => {
-    return (breakpointWidth: BreakpointWidth) =>
-      resolve(breakpointWidth, getGlobalAnalyticsObject('Default Action'));
+    return (breakpointWidth: BreakpointWidth) => resolve(breakpointWidth);
```

Comment:
I am sure we won't ever track this 😄  Can just as well use createStandardAction here.

Reply:
That's done